resources/live_visitors.go

```diff
@@ -13,15 +13,5 @@ func LiveVisitors() *schema.Table {
 		Resolver:  fetchLiveVisitors,
 		Transform: transformers.TransformWithStruct(rest.LiveVisitors{}),
 		Multiplex: client.ProjectMultiplex,
-		Columns: []schema.Column{
-			{
-				Name:     "project_id",
```

Comment:
Removing as https://docs.swetrix.com/statistics-api#get-v1loglivevisitors doesn't return a `project_id` or any unique key in the response